kubernetes/invoker/invoker.yml

```diff
@@ -70,6 +70,12 @@ spec:
           - name: "INVOKER_INSTANCES"
             value: "1"
 
+          # Invoker assigned name. Derived from hostname
+          - name: "INVOKER_NAME"
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
```

Comment:
Would `metadata.name` be a better fieldPath? Couldn't there be multiple invokers on a node? On minikube, for example, `spec.nodeName` returns 'localhost' where `metadata.name` returns `invoker-0`.

Reply:
yep.  sorry, I grabbed this hunk from #55 without thinking about it too carefully and we  merged with minimal review to get the travis build passing again.  I'll submit a fix.